rubin_sim/maf/metrics/summary_metrics.py

```diff
@@ -199,7 +199,7 @@ def __init__(
         lmin=100.0,
         lmax=300.0,
         remove_dipole=True,
-        mask_val=hp.UNSEEN,
+        mask_val=np.nan,
```

Comment:
Later in this metric though, I think you might need to do something like 
data = np.where(np.isnan(data_slice[self.col_name]), hp.UNSEEN, data_slice[self.col_name])
and then use data in the hp.anafast call. 
(since data_slice[self.col_name] here is actually the metric values, which will have been filled with the mask value). 
I thought at some point we were calculating the "TotalPower" summary metric, for desc? But maybe it's not been used for a while.